dags/dap_collector_ppa_dev.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+DOCS = """
+### PPA Dev DAP Collector
+
+#### Description
+
+Runs a Docker image that collects PPA Dev Environment data from a DAP (Distributed Aggregation Protocol) leader and stores it in BigQuery.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/dap-collector-ppa-dev)
+
+This DAG requires following variables to be defined in Airflow:
+* dap_ppa_dev_auth_token
+* dap_ppa_dev_hpke_private_key
+* dap_ppa_dev_task_config_url
+* dap_ppa_dev_ad_config_url
+
+This job is under active development, occasional failures are expected.
+
+#### Owner
+
+bbirdsong@mozilla.com
+"""
+
+default_args = {
+    "owner": "bbirdsong@mozilla.com",
+    "email": ["ads-eng@mozilla.com", "bbirdsong@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2024, 4, 19),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=15),
```

Comment:
Retry delay is the same as schedule interval so you'll have two running concurrently if it fails.  Would it make sense to wait for the next run instead of retrying?  Or is the continuity important?

Reply:
That makes sense. I copied this from another job and did not evaluate the retry logic.

If a job fails, I do not actually think a retry or subsequent run would work, because the job depends on the {{ ts }} variable of the dag run, and only goes back every 15 minutes.

I think what I can do is remove the retry and run the script manually if a job fails.